sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_generated/aio/operations_async/_digital_twin_models_operations_async.py

```diff
@@ -63,29 +63,26 @@ async def add(
 
           * ModelAlreadyExists - The model provided already exists.
 
-        :param models: An array of models to add.
-        :type models: list[object]
+        :param dtdl_models: An array of models to add.
+        :type dtdl_models: list[object]
         :param digital_twin_models_add_options: Parameter group.
         :type digital_twin_models_add_options: ~azure.digitaltwins.core.models.DigitalTwinModelsAddOptions
         :keyword callable cls: A custom type or function that will be passed the direct response
         :return: list of DigitalTwinsModelData, or the result of cls(response)
         :rtype: list[~azure.digitaltwins.core.models.DigitalTwinsModelData]
         :raises: ~azure.core.exceptions.HttpResponseError
         """
-        cls = kwargs.pop('cls', None)  # type: ClsType[List["_models.DigitalTwinsModelData"]]
-        error_map = {
-            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
-        }
+        cls = kwargs.pop('cls', None)  # type: ClsType[List["models.DigitalTwinsModelData"]]
+        error_map = {404: ResourceNotFoundError, 409: ResourceExistsError}
```

Comment:
This comment refers to multiple places: for some reason, we are no longer including 401 here. Do we know why?